compiler/runtime/OMRCodeCacheManager.cpp

```diff
@@ -1033,6 +1033,11 @@ OMR::CodeCacheManager::decreaseFreeSpaceInCodeCacheRepository(size_t size)
       }
    }
 
+bool
+OMR::CodeCacheManager::isAvailableMemorySafeForAllocation(size_t & requestedCodeCacheSize)
+   {
+   return true;
```

Comment:
I would use a comment here indicating that downstream projects can provide a real implementation here.

Reply:
Provided the comments in the method declaration.